app/validation/validate_schema_text.py

```diff
@@ -0,0 +1,43 @@
+import re
+from jsonpointer import resolve_pointer
+from eq_translations.utils import find_pointers_to
+from eq_translations.survey_schema import SurveySchema
+
+
+def get_pointer_keys():
+    pointer_keys = [
+        'show_guidance',
+        'hide_guidance',
+        'description',
+        'legal_basis',
+        'playback',
+        'item_title',
+        'label',
+        'add_link_text',
+        'empty_list_text',
+        'text',
+        'title',
+    ]
+
+    return pointer_keys
+
+
+def validate_json_schema_text(json_schema):
+    pointers = []
+    errors = []
+
+    for key in get_pointer_keys():
```

Comment:
```suggestion
    pointers = [find_pointers_to(json_schema, key) for key in pointer_keys] + schema_object.get_list_pointers()
```

Reply:
I don't think I am able to make this change as I need the return to be a flattened list of strings.

Reply:
No longer required after latest refactor